types/dec_coin.go

```diff
@@ -208,7 +231,7 @@ func (coins DecCoins) TruncateDecimal() (truncatedCoins Coins, changeCoins DecCo
 //
 // CONTRACT: Add will never return Coins where one Coin has a non-positive
 // amount. In otherwords, IsValid will always return true.
-func (coins DecCoins) Add(coinsB DecCoins) DecCoins {
+func (coins DecCoins) Add(coinsB ...DecCoin) DecCoins {
```

Comment:
This is different than the coins variant which takes a slice literal. I'd rather be consistent here and accept the slice literal instead of variadic args. What was the rationale for this @fedekunze?

Reply:
I will wait for the response of @fedekunze and based on that I will do the necessary changes.

Reply:
> What was the rationale for this @fedekunze?

see https://github.com/cosmos/cosmos-sdk/pull/5449#discussion_r361477102

Reply:
I don't see the rationale there. The API is still inconsistent. Change this method or also change `Coins#Add`.

Reply:
If I can have my own opinion, I would change the `Coins#Add` because the variadic args opens up more clear ways to pass one or more data or slice of data.

Reply:
@alexanderbez I changed Coins#Add, also marked in the API Breaking section. If you wanted to move to the other direction and revert the changes on the DecCoins#Add please let me know.